src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2016, Ford Motor Company
```

Comment:
@dcherniev fixed in https://github.com/smartdevicelink/sdl_core/pull/1234/commits/817018f15bcc9a280711589e063f3a1a262e115b